.lintstagedrc

```diff
@@ -1,3 +1,3 @@
 {
-  "**/*": "npm run format"
+  "**/*": ["eslint --fix", "prettier --write"]
```

Comment:
I think I recall that there's a reason why we use `next lint`, but I wonder now if it was by any particular reason.

Before we make this change, @bmuenzenmeyer could you check if `next lint` adds some other linting configuration?

I think if I recall correctly, I was using `next lint` because Next.js already knows all the files it needs to lint. But on `lint-staged` it makes more sense to have a direct eslint call, as we want to lint the staged files and we know which files we want to lint.

Reply:
I tried this and it does not work:

```
[STARTED] Preparing lint-staged...
[SUCCESS] Preparing lint-staged...
[STARTED] Running tasks for staged files...
[STARTED] .lintstagedrc — 1 file
[STARTED] **/* — 1 file
[STARTED] next lint --fix
[FAILED] next lint --fix [FAILED]
[FAILED] next lint --fix [FAILED]
[FAILED] next lint --fix [FAILED]
[STARTED] Applying modifications from tasks...
[SKIPPED] Skipped because of errors from tasks.
[STARTED] Reverting to original state because of errors...
[SUCCESS] Reverting to original state because of errors...
[STARTED] Cleaning up temporary files...
[SUCCESS] Cleaning up temporary files...

✖ next lint --fix:
(node:34940) ExperimentalWarning: Importing JSON modules is an experimental feature and might change at any time
(Use `node --trace-warnings ...` to show where the warning was created)
- warn You have enabled experimental features (largePageDataBytes, outputFileTracingExcludes) in next.config.mjs.
- warn Experimental features are not covered by semver, and may cause unexpected or broken application behavior. Use at your own risk.

C:\src\nodejsorg-3\node_modules\next\dist\lib\find-pages-dir.js:94
        throw new Error("> Couldn't find any `pages` or `app` directory. Please create one under the project root");
              ^

Error: > Couldn't find any `pages` or `app` directory. Please create one under the project root   
    at findPagesDir (C:\src\nodejsorg-3\node_modules\next\dist\lib\find-pages-dir.js:94:15)       
    at nextLint (C:\src\nodejsorg-3\node_modules\next\dist\cli\next-lint.js:206:67)

Node.js v18.13.0
husky - pre-commit hook exited with code 1 (error)
```

This behavior is exactly as reported elsewhere: https://stackoverflow.com/questions/68419192/nextjs-linter-with-lint-staged-not-working

Searching for this further, there is an avalanche of folks directly using `eslint`.

Reply:
Let's for now merge this as it is, and if worth investigate more later :)